pset-functions.js

```diff
@@ -135,7 +160,22 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+function scrabbleScore(word) {
+  const letters = word.split(' ')
+  const wordCount = {}
+
+  letters.forEach((letter) => {
+    if (wordCount[letter] === undefined) {
+      wordCount[letter] = 1
+    } else {
+      wordCount[letter] += 1
+    }
+    // console.log(wordCount)
+  })
+  return wordCount
+}
+
+console.log(scrabbleScore('cabbage'))
 // Add your code above this line
```

Comment:
@OcielC Good work coming up with this solution here. for this assignment, it needed to be done slightly different. This solution would work well for problem 3 and this problem here you had to compare each letter of the string with a letter that will give you a certain amount of points per letter.

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L174)